tests/integration/test_hive_query/test.py

```diff
@@ -109,24 +120,19 @@ def test_parquet_groupby_with_cache(started_cluster):
     assert result == expected_result
 def test_cache_read_bytes(started_cluster):
     node = started_cluster.instances['h0_0_0']
-    node.query("set input_format_parquet_allow_missing_columns = true")
     result = node.query("""
-    DROP TABLE IF EXISTS default.demo_parquet;
-    CREATE TABLE default.demo_parquet (`id` Nullable(String), `score` Nullable(Int32), `day` Nullable(String)) ENGINE = Hive('thrift://hivetest:9083', 'test', 'demo') PARTITION BY(day)
+    CREATE TABLE IF NOT EXSISTS default.demo_parquet_1 (`id` Nullable(String), `score` Nullable(Int32), `day` Nullable(String)) ENGINE = Hive('thrift://hivetest:9083', 'test', 'demo') PARTITION BY(day)
             """)
-    result = node.query("""
-    SELECT day, count(*) FROM default.demo_parquet group by day order by day
+    test_passed = False
+    for i in range(10):
+        result = node.query("""
+    SELECT day, count(*) FROM default.demo_parquet_1 group by day order by day settings input_format_parquet_allow_missing_columns = true
             """)
-    result = node.query("""
-    SELECT day, count(*) FROM default.demo_parquet group by day order by day
-            """)
-    expected_result = """2021-11-01	1
-2021-11-05	2
-2021-11-11	1
-2021-11-16	2
-"""
-    time.sleep(120)
-    assert result == expected_result
-    result = node.query("select sum(ProfileEvent_ExternalDataSourceLocalCacheReadBytes)  from system.metric_log where ProfileEvent_ExternalDataSourceLocalCacheReadBytes > 0")
-    logging.info("Read bytes from cache:{}".format(result))
-    assert result.strip() != '0'
+        result = node.query("select sum(ProfileEvent_ExternalDataSourceLocalCacheReadBytes)  from system.metric_log where ProfileEvent_ExternalDataSourceLocalCacheReadBytes > 0")
+        if result.strip() == '0':
+            logging.info("ProfileEvent_ExternalDataSourceLocalCacheReadBytes == 0")
+            time.sleep(60)
```

Comment:
1 minute sleep in test seems too much, is it really needed?

Reply:
change to 10s